com.microsoft.mrtk.input/Simulation/Devices/ControllerSimulationSettings.cs

```diff
@@ -46,31 +48,72 @@ public ControllerAnchorPoint AnchorPoint
         }
 
         [SerializeField]
-        [Tooltip("The input action used to indicate that the hand's neutral pose should be changed.")]
-        private InputActionReference changeNeutralPose;
+        [FormerlySerializedAs("changeNeutralPose")]
+        [Tooltip("The input action used to toggle between using the default or Secondary Handshape settings.")]
+        private InputActionReference toggleSecondaryHandshapes;
```

Comment:
nit: can this just be `toggleHandshapes` to match the original name that didn't reference `secondary`? It seems slightly misleading, since it toggles between the default and the secondary, not between "secondary handshapes".

Reply:
ehh, I guess I could read it as "toggle secondary handshapes on/off" as well, which is fine. I was initially wanting to read it as "toggle between handshapes"

Reply:
I don't want to call it `toggleHandshapes` because it could indicate that we're toggling between neutral and trigger handshapes. In an idea world we have "handshape sets" which the user can toggle between but that would make the input simulator controls more confusing without adding that much utility I think. Thus I settled with adding the "secondary" moniker.